web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
Can I use the `function` definition? I think it would be so much nicer to read to be able to define functions after they're being used:
```
angular.module(module.moduleName, [ProjectConfig.moduleName])
    .service([
        '$http',
        ProjectConfig.componentName,
        API
    ]);

export function API ($http, ProjectConfig) {
    return {
        getBaseUrl: function () {
            const endpoint = 'forms';
            const apiRoot = ProjectConfig.getSettings().apiRoot;
            return `${apiRoot}/${endpoint}`;
        },
        getList: function () {
            const url = this.getBaseUrl();

            return $http.get(url)
                .then((response) => response.data);
        },
        getForm: function (url) {
            return $http.get(url)
                .then((response) => response.data);
        }
    }
};
```